broker-daemon/models/market-event.js

```diff
@@ -63,24 +65,30 @@ class MarketEvent {
     const baseCommonAmount = Big(this.payload.baseAmount).div(baseCurrencyConfig.quantumsPerCommon)
     const counterCommonAmount = Big(this.payload.counterAmount).div(counterCurrencyConfig.quantumsPerCommon)
 
+    console.log(this.payload)
+    console.log(this.payload)
+
+    if (baseCommonAmount.eq(0)) {
+      return Big(0).toString()
```

Comment:
Did you figure out this issue?

Reply:
This was only a dev line because of an out-of-date orderbook. We are all good